packages/firestore/src/local/shared_client_state.ts

```diff
@@ -409,8 +538,27 @@ export class WebStorageSharedClientState implements SharedClientState {
     );
   }
 
+  private persistMutationState(
+    batchId: BatchId,
+    state: MutationBatchState,
+    error?: FirestoreError
+  ) {
+    const mutationState = new RemoteMutationBatch(
+      this.user,
+      batchId,
+      state,
+      error
+    );
+
+    const mutationKey = `${MUTATION_BATCH_KEY_PREFIX}_${this.persistenceKey}_${
+      this.user.uid
+    }_${batchId}`;
```

Comment:
FWIW- This is the kind of prettier formatting where I'd be tempted to undo prettier's change and instead do:

```
    const mutationKey = `${MUTATION_BATCH_KEY_PREFIX}_${this.persistenceKey}_` +
        `${this.user.uid}_${batchId}`;
```

Not asking you to change it... just calling it out for context on the prettier stuff. :-)